.github/workflows/build.yml

```diff
@@ -189,3 +189,16 @@ jobs:
           MC_TWITTER_ACCESS_TOKEN_SECRET: ${{ secrets.MC_TWITTER_ACCESS_TOKEN_SECRET }}
           MC_TWITTER_CONSUMER_KEY: ${{ secrets.MC_TWITTER_CONSUMER_KEY }}
           MC_TWITTER_CONSUMER_SECRET: ${{ secrets.MC_TWITTER_CONSUMER_SECRET }}
+
+      - name: List test failures and errors
+        run: echo -e 'Test failures:\n' && awk '/FAILURES/,/short/' /home/runner/runners/*/_diag/pages/*.log && echo -e '\n\nErrors:\n' && awk '/non-zero/,/status 1/' /home/runner/runners/*/_diag/pages/*.log
+        if: always()
+      
+          # - name: Setup tmate session
+      #   uses: mxschmitt/action-tmate@v3
+      #   if: always()
+      #   with:
+      #     limit-access-to-actor: true
+
+      # ^uncomment this if you need to debug the GitHub actions build
+      # TODO: link to new README section once the related branch is merged
```

Comment:
Can you revert this before merging? This is a part of https://github.com/mediacloud/backend/pull/765 I think.

Reply:
for sure, i didn't intend to leave it in for the final PR—still trying to work out the issue of the GHA build not working